examples/latent_sde.py

```diff
@@ -28,6 +28,7 @@
 
 from examples import utils
 from torchsde import sdeint, sdeint_adjoint, SDEIto, BrownianInterval
+from torchsde._core.misc import _stable_division
```

Comment:
Let's not put `_stable_division` in core, but directly in this file.

Reply:
Sure!